src/Common/Navbar/NavbarOotd/NavbarOotd.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable jsx-a11y/alt-text */
+import React, { Component } from 'react';
+import './NavbarOotd.scss';
+
+class NavbarOotd extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOotdBorder: false,
+      isStoreBorder: false,
+    }
+  }
+  handleClickOotd = (e) => {
+    e.defaultProps;
+    return (
+      this.setState({
+        isOotdBorder: true,
+        isStoreBorder: false,
+      })
+    )
+  }
+
+  handleClickStore = (e) => {
+    e.defaultProps;
+
+    return (
+      this.setState({
+        isOotdBorder: false,
+        isStoreBorder: true,
+      })
+    )
+  }
+
+  render() {
+    const { isOotdBorder, isStoreBorder } = this.state;
+    return (
+      <>
+      <div className="navBarWrapper">
+        <nav className="topNavBar">
+          <div className="leftNavBar">
+            <a className="navLogo" href="/">
+              <img src='/images/navLogo.png' />
+            </a>
+            <div className="leftMenuWrapper">
+              <ul className="leftMenuList">
+                <li value="feed"
+                    className="leftMenu">
+                  <span  className={isOotdBorder === true? "leftMenuOotdAct":"leftMenuOotd"} onClick={this.handleClickOotd}>#OOTD</span>
```

Comment:
className의 동적사용!! :) 굿굿! 
boolean을 이용해서 아래와 같이 작성해주실 수 있습니다. 
```suggestion
                  <span  className={isOotdBorder ? "leftMenuOotdAct":"leftMenuOotd"} onClick={this.handleClickOotd}>#OOTD</span>
```

Reply:
boolean 유용하게 썼습니다!! 감사합니다 :)